lib/cocina/models/validators/description_values_validator.rb

```diff
@@ -64,6 +67,18 @@ def validate_values_for_multiples(hash, path)
           error_paths_multiple << path_to_s(path)
         end
 
+        def validate_title_type(hash, path)
+          # only apply to title.structuredValue or relatedResource.title.structuredValue with value
+          return unless hash[:value] && (path.first == :title || related_resource_title?(path)) && path.include?(:structuredValue)
+
+          # if there is a "value" key, make sure there is also a "type" key, only for title.structuredValue
+          error_paths_missing_title_type << path_to_s(path) unless hash[:type]
+        end
+
+        def related_resource_title?(path)
+          path.first == :relatedResource && path.third == :title
```

Comment:
what is second?   structuredValue or parallelValue or something?  It's theoretically possible to have structuredValues inside parallelValues ...

Reply:
Is there an easy way to model that scenario in another spec and see what happens?  Do you think that should block this fix?

Reply:
When I talked this through with Justin, we went down the path of keeping this in structuredValue only. Not further nesting or other situations. 

The path it's looking for would be `[:title, 0, :structuredValue, 0]` in order to validate the presence of a `type`. in the hash. 

Is there a way you'd recommend we do this? I don't doubt there's something I'm not understanding here.

Reply:
@ndushay and I took a look at this and I'll add in the case to handle `title.parallelValue.structuredValue`. @arcadiafalcone says: this would likely be coming from a MARC record, so with Google Books it's not that uncommon.

Reply:
@ndushay see added specs to cover `title.parallelValue.structuredValue` and `relatedResource.title.parallelValue.structuredValue` situations. Because the logic is to check for `structuredValue`  in the path, but not directly as a descendant of `title`, there is room for a `parallelValue` to exist in the path. I think this is OK for titles?